lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
```

Comment:
@ziafazal My first thought is that you should move your file to:

    lms/static/js/spec/instructor_dashboard

I think the coffee directory is not on the RequireJS load path.

An example of a file you can look for is ecommerce_spec.js from inside that directory. It is registered in js/spec/main.js.